Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Start with Python base image
+FROM python:3.9-slim
+
+# Set the working directory
+WORKDIR /app
+
+# Copy necessary files
+COPY . /app
+
+# Install dependencies
+RUN pip install wallycore
+
+# Generate a self-signed certificate (optional)
+RUN openssl req -new -x509 -keyout server.pem -out server.pem -days 3650 -nodes -subj "/CN=localhost"
```

Comment:
Consider a bind mount or Docker secret instead of embedding the certificate directly into the container.

Reply:
Done

Reply:
Actually, why not get rid of the need to use a self-signed certificate at all?

If protection becomes necessary, we could use a reverse-proxy, like nginx.

I tested removing the following line and it still seemed to work
```python
server.socket = ssl.wrap_socket(server.socket, certfile='./key_data/server.pem', server_side=True)
```

Reply:
@Filiprogrammer can you please comment on this?

Reply:
Once you get to step 3/4 and you click "Request Camera Permission" you will get the following error message when not using HTTPS on a non-localhost connection:
"Camera access is only supported in secure context like https or localhost."

This is the reason for the TLS certificate. The camera cannot be used without it.
As you mentioned one could use a reverse-proxy, so I just added a `--no-tls` command line option to disable TLS.

Reply:
Understood, thank you for the clarification.

Is there anything that I need to address for you to accept this pull request?

Reply:
A default certificate is still stored inside the Docker image, which is generally considered bad practice.
I believe it should be on the user to generate the certificate manually if they need it. _(Or use reverse proxy like Caddy with automatic certificate generation.)_

Reply:
> I believe it should be on the user to generate the certificate manually if they need it.

Sure, agreed.

I removed the certificate generation inside the Docker image altogether as I'm passing `--no-tls` in `docker-compose.yml`